src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -37,13 +37,13 @@ class IngestUploadWdg(BaseRefreshWdg):
         'parent_key': 'Parent search key to relate create sobject to',
         'ingest_data_view': 'Specify a ingest data view, defaults to edit',
         'extra_data': 'Extra data (JSON) to be added to created sobjects',
-        'oncomplete_script_path': 'Script to be run on a finished ingest'
+        'oncomplete_script_path': 'Script to be run on a finished ingest',
+        'update_mode': 'Ingest files with same name as another file sObject.'
```

Comment:
should list the possible mode here.. true|false
Can u rephrase this description so it's easier to understand